src/receiving.rs

```diff
@@ -297,6 +309,16 @@ impl Receiver {
         }
     }
 
+    pub fn get_change_address(&self) -> String {
```

Comment:
I assume the caller will use this 'change address' and send to it as if it were a normal silent payment address? That's kind of elegant actually, although we should be careful that the caller never uses this address for anything else. Would just adding a bit in the documentation be enough?

Reply:
That's what I was thinking indeed, and while your concern is valid I also think that's the responsibility of the wallet implementer and more an UX issue. I think that in a sp wallet there should be no way for the user to see this address so that it can't be misused.

Reply:
I think we should add some documentation to this function, then it can be merged